paypal-commercetools-extension/src/service/customers.service.ts

```diff
@@ -167,6 +167,7 @@ export const handleGetPaymentTokensRequest = async (
   );
   try {
     const response = await getPaymentTokens(customerId);
+    response.payment_tokens = response.payment_tokens?.filter((paymentToken) => paymentToken.payment_source && !('apple_pay' in paymentToken.payment_source));
```

Comment:
response.payment_tokens = response.payment_tokens?.filter(({payment_source}) => payment_source && !('apple_pay' in payment_source));